packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -408,11 +474,24 @@ export class ProposedTransformationExplorer {
             diffModel.saveChanges()
             telemetry.ui_click.emit({ elementId: 'transformationHub_acceptChanges' })
             void vscode.window.showInformationMessage(CodeWhispererConstants.changesAppliedNotification)
-            transformByQState.getChatControllers()?.transformationFinished.fire({
-                message: CodeWhispererConstants.changesAppliedChatMessage,
-                tabID: ChatSessionManager.Instance.getSession().tabID,
-            })
-            await reset()
+            if (!diffModel.currentPatchIndex) {
```

Comment:
Would prefer `if(diffModel.currentPatchIndex === 0)` plus a comment stating that we do this so that the changesAppliedChatMessage is only sent to user when they accept the first `diff.patch`